api/storage_service.cc

```diff
@@ -508,10 +508,9 @@ void set_storage_service(http_context& ctx, routes& r, sharded<service::storage_
         return scylla_version();
     });
 
-    ctx.db.local().get_builtin_routine_registry().register_routine("scylla_version",
-            db::make_single_value_routine<sstring>("scylla_version", [](db::builtin_routine::parameter_map&& par) {
-        return make_ready_future<sstring>(scylla_version());
-    }));
+    db::add_system_function("scylla_version", [] { 
```

Comment:
Please remove the scylla_version routine from the previous patch if you don't intend it to survive.